ui/src/components/HydraRawRdfForm.vue

```diff
@@ -5,6 +5,7 @@
       v-if="shape"
       format="text/turtle"
       :quads.prop="initQuads"
+      :prefixes="editorPrefixes"
```

Comment:
Does this vue stringify the array?

Reply:
Probably not, why?

Reply:
https://zazuko.github.io/rdfjs-elements/rdf-editor/

> `prefixes`: a comma-separated list of prefixes to use for serializing.

Reply:
Funny, it seemed to work 🤔

Reply:
Good, so maybe that is what vue does. Native `Array#toString()` simply comma-joins its elements 👌